src/Evaluators/common.jl

```diff
@@ -112,7 +111,9 @@ function _inf_pr(nlp::AbstractNLPEvaluator, cons)
     return max(err_inf, err_sup)
 end
 
-# Scaler utils
+#=
+    SCALER
+=#
 abstract type AbstractScaler end
```

Comment:
`Inf` norm is called in here. It might be worthwhile implementing a kernel for that using `abs.(x)` and `reduce(max, x)`. Not sure though.

Reply:
It makes sense to create a new kernel for that, indeed. In fact, we have already implemented that kernel in #181 : 
https://github.com/exanauts/ExaPF.jl/blob/develop/src/architectures.jl#L15